.github/workflows/builder_maven_slsa3.yml

```diff
@@ -22,42 +22,56 @@ on:
         required: false
         type: boolean
         default: false
-      artifact-list:
-        description: "Comma-separated list of paths to artifacts to be released. Use PROJECT_VERSION for the release version, for example: 'artifact-PROJECT_VERSION-sources.jar'"
-        type: string
-        required: true
       jdk-version:
         description: "The JDK version"
         required: false
         default: 17
         type: number
+
+    outputs:
+      provenance-name:
+        description: "The file name of the attestation upload artifact."
+        # NOTE: This is an "untrusted" value returned from the build.
+        value: "${{ fromJSON(jobs.slsa-run.outputs.build-artifacts-outputs).attestation-name }}"
+
+      provenance-download-name:
+        description: "The name of the provenance attestation uploaded to the workflow run."
+        value: "${{ jobs.slsa-run.outputs.attestations-download-name }}"
+
+      provenance-download-sha256:
+        description: "The sha256 of the provenance attestation uploaded to the workflow run."
+        value: "${{ jobs.slsa-run.outputs.attestations-download-sha256 }}"
+
+      target-download-sha256:
+        description: "The sha256 of the target uploaded to the workflow run."
```

Comment:
I suppose "target" is the commonly-used terminology in Java for built artifacts? If not, add an example.

Reply:
Yes. Maven creates a `target` directory for the built artifacts.